iceoryx_examples/icehello/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Build icehello example
+cmake_minimum_required(VERSION 3.5)
+project(example_icehello)
+
+include(GNUInstallDirs)
+
+find_package(iceoryx_posh CONFIG REQUIRED)
+find_package(iceoryx_utils CONFIG REQUIRED)
+
+get_target_property(ICEORYX_CXX_STANDARD iceoryx_posh::iceoryx_posh CXX_STANDARD)
+if ( NOT ICEORYX_CXX_STANDARD )
+    include(IceoryxPlatform)
```

Comment:
We have to always include this otherwise the `${ICEORYX_WARNINGS}` are empty and we have to fix warnings again in examples.

So could you please remove the `if` around `include(IceoryxPlatform)`

Reply:
Take a look at: https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/icedelivery/CMakeLists.txt in master as a reference

Reply:
Done